src/api.js

```diff
@@ -1,8 +1,15 @@
 import { useState, useEffect } from 'react';
 import axios from 'axios';
 
-export async function getArticles() {
-    const { data: {articles} } = await axios.get('https://nc-news-th0a.onrender.com/api/articles');
+export async function getArticles(topic) {
+
+    let url = `https://nc-news-th0a.onrender.com/api/articles`;
+
+    if(topic !== undefined && topic !== null) {
+        url += `?topic=${topic}`;
```

Comment:
If you take a look at Axios Params, you wouldn't need to have this logic in here 👍